frontend/plan/components/selector/Selector.tsx

```diff
@@ -47,27 +62,10 @@ const Selector: FunctionComponent<SelectorProps> = ({
     const isLoading =
         isSearchingCourseInfo || (isLoadingCourseInfo && !isExpanded);
 
-    const loadingIndicator = (
-        <div
-            className="button is-loading"
-            style={{
-                height: "100%",
-                width: "100%",
-                border: "none",
-                fontSize: "3rem",
-            }}
-        />
-    );
+    const loadingIndicator = <Loading className="button is-loading" />;
```

Comment:
now that it's a one-line component, could it be inlined? or is `loadingIndicator` mentioned in multiple places?

I also wonder if the `is-loading` class could be turned into a styled component so we don't have to rely on stylesheets anymore. Maybe factor it out into a common file?

Reply:
`loadingIndicator` is in 2 places, but it's def able to be put in line - would probably be cleaner this way

Reply:
`is-loading` has been pulled out - is this used in several places?  Right now, I just have it in the `Selector.tsx` file, but if it's used across components, I could also pull it into the derived_components file

Reply:
This seems like a useful thing to have in the derived_components even if it's only in one place right now

Reply:
Done!